src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_DateAndTime.vb

```diff
@@ -89,5 +91,35 @@ class c
                 Await state.AssertNoCompletionSession()
             End Using
         End Function
+
+        <WpfTheory, CombinatorialData, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestExample1(showCompletionInArgumentLists As Boolean) As Task
+            Using state = TestStateFactory.CreateCSharpTestState(
+<Document><![CDATA[
+using System;
+class c
+{
+    void goo(DateTime d)
+    {
+        d.ToString("hh:mm:$$");
+    }
+}
+]]></Document>, showCompletionInArgumentLists:=showCompletionInArgumentLists)
+
+                state.SendInvokeCompletionList()
+                state.SendTypeChars("ss")
+                Await state.AssertSelectedCompletionItem("ss", inlineDescription:=FeaturesResources.second_2_digits)
+
+                Dim description = Await state.GetSelectedItemDescriptionAsync()
+                Dim text = description.Text
+
+                If CultureInfo.CurrentCulture.Name <> "e-US" Then
```

Comment:
```suggestion
                If CultureInfo.CurrentCulture.Name <> "en-US" Then
```

?

Reply:
Which this should have caused the test to fail, right? Since in en-US we would have tried to assert both en-US and e-US, and on our Spanish leg it should have _passed_, right? Did the test not run?